tests/models_core/test_magic_wand.py

```diff
@@ -47,6 +48,7 @@ def test_magic_wand(
     dense_outputs = dense_model.generate_greedy_logprobs(
         example_prompts, max_tokens, num_logprobs)
     del dense_model
+    gc.collect()
```

Comment:
should you also `gc.collect()` the `sparse_model` down below too?

Reply:
Script is over by then, so not needed. We just need to make sure the dense model is cleaned up before we start the sparse model

Reply:
approved, but wondering if we should try to encapsulate cleanup in a helper function. even if "gc.collect()" isn't always needed it might be good practice.

Reply:
what about for the next test, though?  are we guaranteed that everything is cleaned up by then?

Reply:
yep